lib/parser/models.ex

```diff
@@ -221,6 +232,29 @@ end
       end
     end
 
+    defmodule TEnumValue do
+      @moduledoc """
+      A reference to an enum value
+      For example, in a constant or default value.
+
+         const string DEFAULT_WEATHER = Weather.SUNNY;
+     """
+      @type t :: %TEnumValue{enum_name: atom, enum_value: atom, type: atom}
+      defstruct enum_name: nil, enum_value: nil, type: nil
```

Comment:
Nearby structs have fields like `name` and `value` or `values`. Would that make sense here, for consistency, rather than `enum_name` and `enum_value`?